qa/tests/features/new/manual/filter-refiner/Range Filter/Filter Refiner-Range Filter.feature

```diff
@@ -0,0 +1,127 @@
+﻿@regression
+Feature: Filter Refiner, Filter by range attribute
+
+Background: Given Anfisa main page was opened
+	When user clicks on "xl_PGP3140_wgs_NIST-4_2" dataset
+	And clicks on "Open" drop-down
+	And selects Filter Refiner
+	Then "Filter Refiner" for the "xl_PGP3140_wgs_NIST-4_2" dataset should be opened
+
+Scenario Outline: 01 Filter by range, middle - XL dataset
+	When user clicks on <Attribute Name>
+	And enters the minimum value of range <Minimum Value>
+	And changes "<" sign with "≤" next to the minimum value
+	And enters the maximum value of range <Maximum Value>
+	And clicks the "+ Add Attribute" button to apply the filter
+	Then filter should be applied
+	And the range <Minimum Value> <= <Attribute Name> <= <Maximum Value> should be displayed in the right part of the screen
+	And number of variants should be equal to <Variants Number>
+	And the "View Variants" dialog can be opened
+	And the "Full List" view should be disabled
+
+Examples:
+	| Attribute Name               | Minimum Value | Maximum Value | Total variants |
+	| Num_Samples                  | 1             | 2             | 3281689        |
+	| Num_Genes                    | 5             | 10            | 652            |
+	| Start_Pos                    | 25200         | 255200        | 5524           |
+	| gnomAD_PopMax_AN             | 20            | 100           | 4632           |
+	| Number_submitters            | 2             | 16            | 5165           |
+	| Min_GQ                       | 0             | 750           | 5591394        |
+	| Number_of_clinvar_submitters | 2             | 22            | 5418           |
+	| Severity                     | 1             | 2             | 52248          |
+
+Scenario Outline: 02 Filter by range, boundary values - XL dataset
```

Comment:
Add please another case: when user doesn't input left or right value it is counted as minimum for the left value and maximum for the right